### ISSUE TITLE
Running tut with fatal-warnings + warn-unused-imports breaks import blocks

### ISSUE THREAD

**Issue by ChrisCoffey** [tpolecat/tut#105]
### Issue
I have a project I'd like to run with both `-Xfatal-warnings` & `-Ywarn-unused-imports`. If I use tut to evaluate the following block it fails to compile:

```
` ``tut
import fs2.Stream

val s = Stream((0 until 100): _*)
s.runLog.run.run
` ``
``` 

### Expected Behavior
I know tut simulates a repl, so the line-by-line behavior isn't entirely unexpected, but I would love if `tut:book` actually evaluated code as a block rather than line-by-line. 

It's obviously easy to work around this by disabling one of the compiler flags.

**Comment by Jentsch** [comment]
At least as a workaround you could use:

```sbt
tutScalacOptions := { 
  val testOptions = scalacOptions.in(test).value
  val unwantedOptions = Set("-Ywarn-unused-import", "-Xfatal-warnings")
  testOptions.filterNot(unwantedOptions)
}
```

But maybe this is a good default and could be inserted at [Plugin.scala Line 50](https://github.com/tpolecat/tut/blob/425c0b5879f8fbed8635be9a39490656b11e1c31/plugin/src/main/scala/tut/Plugin.scala#L50)?

**Comment by tpolecat** [comment]
@Jentsch that seems reasonable; the issue isn't with tut specifically, it's just that the REPL can't cope with those compiler options.

**Comment by Jentsch** [comment]
Yes it's the REPL. I copied it from an other project, where I did the same for the console. [See AutoLifts Repl](https://github.com/wheaties/AutoLifts/pull/58/files#diff-877df953c36503c1a0bf7e1e2053137fR17)

**Comment by Jentsch** [comment]
Btw: why has tut its own ```tutScalacOptions``` and don't use something like ```scalacOptions in tut```?

**Comment by tpolecat** [comment]
Because I don't know how to do the second one.
